<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>GHI CHÚ</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_tongtien">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>NHÂN VIÊN XỬ LÝ</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_nvxl">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_them">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(255, 255, 215); color: rgb(149, 106, 214); border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>THÊM</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string>color: rgb(149, 106, 214);</string>
   </property>
   <property name="text">
    <string>ĐƠN HÀNG MỚI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>TRẠNG THÁI</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_thanhtoan">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>THANH TOÁN</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_trangthai">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>TỔNG TIỀN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>MÃ KHÁCH HÀNG</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_maKH">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_note">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_timKH">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(255, 255, 215); color: rgb(149, 106, 214); border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>TÌM</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
